libs/libcoap/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libcoap
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/obgm/libcoap
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)
+PKG_SOURCE_VERSION:=683337890e6fbab1eaa0ef0f2038194fb62b7fa2
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.bz2
+
+PKG_MAINTAINER:=Anton Glukhov <anton.a.glukhov@gmail.com>
+PKG_LICENSE:=GPL-2.0+
+PKG_LICENSE_FILE:=LICENSE.GPL
```

Comment:
Actually, it's dual licensed with BSD.  That's a _very_ important note.  (A very very odd choice, but ok, so be it)  I SPDX says this is how to do that...

```
How does one represent a file or package that is dual licensed (i.e., a license choice)?
SPDX license information can be represented using conjunctive or disjunctive regular expressions.  For example, a file that is dual licensed under either the GPL-2.0 or MIT would be represented using the following disjunctive expression: (GPL-2.0 or MIT). 
```